nucypher/blockchain/eth/actors.py

```diff
@@ -858,10 +862,35 @@ def locked_tokens(self, periods: int = 0) -> NU:
 
     @property
     def current_stake(self) -> NU:
+        """The total number of staked tokens, i.e., tokens locked in the current period."""
+        return self.locked_tokens(periods=0)
+
+    @property
+    def active_stakes(self) -> Generator[Stake, None, None]:
+        """Returns active stakes for this staker."""
+        stakes = (stake for stake in self.stakes if stake.is_active)
+        return stakes
+
+    @property
+    def sorted_stakes(self) -> List[Stake]:
+        """Returns a list of active stakes sorted by account wallet index."""
+        stakes = sorted(self.active_stakes, key=lambda s: s.address_index_ordering_key)
+        return stakes
+
+    def __filter_divisible_stakes(self, stake: Stake) -> bool:
         """
-        The total number of staked tokens, i.e., tokens locked in the current period.
+        Helper function for use as a filtration predicate when determining
+        what constitutes a divisible stake amongst a given iterable of stakes.
         """
-        return self.locked_tokens(periods=0)
+        min_divisible_stake_value = self.economics.minimum_allowed_locked * 2
+        result = bool(stake.value >= min_divisible_stake_value)
```

Comment:
Seems a little redundant, comparison is already `bool`.

Reply:
Heh - yeah I have this style habit for bool functions.

Reply:
:heavy_check_mark: